packages/vite/src/node/optimizer/index.ts

```diff
@@ -210,50 +233,110 @@ export async function createOptimizeDepsRun(
   }
 
   const depsCacheDir = getDepsCacheDir(config)
-  const processingCacheDir = getProcessingDepsCacheDir(config)
-
-  const mainHash = getDepHash(root, config)
-
-  const processing = newDepOptimizationProcessing()
-
-  const metadata: DepOptimizationMetadata = {
-    hash: mainHash,
-    browserHash: mainHash,
-    optimized: {},
-    chunks: {},
-    discovered: {}
-  }
 
   if (!force) {
-    let prevData: DepOptimizationMetadata | undefined
+    let cachedMetadata: DepOptimizationMetadata | undefined
     try {
-      const prevDataPath = path.join(depsCacheDir, '_metadata.json')
-      prevData = parseOptimizedDepsMetadata(
-        fs.readFileSync(prevDataPath, 'utf-8'),
+      const cachedMetadataPatah = path.join(depsCacheDir, '_metadata.json')
```

Comment:
Is `Patah` a typo?

Reply:
I should have used `cachedMetadataPatak` there